assignments/assignment01.rb

```diff
@@ -68,7 +95,14 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  # your implementation here
+  chars = s.downcase.chars
+  word = ""
+  chars.each do |c|
```

Comment:
what you are trying to do here is remove non-letter characters from the string. There are much more elegant ways of doing this in ruby. You could use delete, you could use gsub. You don't need to brute-force do it yourself a character at a time.